priv/repo/migrations/20170419210307_create_user.exs

```diff
@@ -0,0 +1,15 @@
+defmodule RemoteRetro.Repo.Migrations.CreateUser do
+  use Ecto.Migration
+
+  def change do
+    create table(:users) do
+      add :google_user_info, :jsonb
```

Comment:
any reason for storing the user attributes as json? wouldn't it be easier to just cast the map to a row in the database, i.e. with a `given_name` column, `family_name` column, and so on?